crypto/cmp/cmp_util.c

```diff
@@ -101,7 +101,7 @@ const char *ossl_cmp_log_parse_metadata(const char *buf,
                     /* no real problem if OPENSSL_strndup() returns NULL */
                     *line = (int)line_number;
                     msg = strchr(p_level, ':') + 1;
-                    if (*msg == ' ')
```

Comment:
strchr() can evaluate to NULL, so `strchr() + 1` could add to a NULL pointer, which is UB?